tests/html_quotations_test.py

```diff
@@ -2,13 +2,11 @@
 
 from __future__ import absolute_import
 
-# noinspection PyUnresolvedReferences
-import re
-
+from .fixtures import REPLY_QUOTATIONS_SHARE_BLOCK, OLK_SRC_BODY_SECTION, REPLY_SEPARATED_BY_HR
```

Comment:
As long as you are updating imports here and there could you please make them absolute.